src/asciidoc/docs/getting-started/advQuickStart.adoc

```diff
@@ -0,0 +1,95 @@
+= Quick start
+:page-slug: adv-quick-start
+:page-description: Instructions to quickly start coding your first project.
+:includespath: ../_includes
+
+
+Welcome to Koji!
+
+This quick start topic guides you through creating your first project in the Koji IDE and developing your first Koji app.
+You must be comfortable with developing web applications.
+
+* If you are new to Koji and have minimal or no web development experience, go through <<quick-start#, the Koji quick-start tutorial for beginners>> instead.
+* If you prefer to work locally, follow the <<work-locally#,Working locally>> instructions.
+
+
+== Prerequisites
+
+* https://withkoji.com/[A Koji account]
+
+
+== Getting started with Koji
+
+. To create a project, fork an existing scaffold or any project that allows forking.
++
+.Step details ...
+[%collapsible]
+====
+.. Go to the https://withkoji.com/create/for-developers[For Developers] page.
+.. Choose a scaffold and click its *VIEW* button.
++
+image::web-dev-scaffold-list.png[List of scaffolds,50%,link={imagesDir}/web-dev-scaffold-list.png]
+.. In the scaffold's detail page, click the ellipses icon ( *...* ) and choose *Fork*.
++
+image::web-react-scaffold-ellipses-fork.png[Ellipses menu then fork,50%,link={imagesDir}/web-react-scaffold-ellipses-fork.png]
+====
+
+. Develop your app.
+.. In the Project Details page, rename your new project, then open it in the Code Editor.
++
+.Image: Project Details ...
+[%collapsible]
+====
+image::project-details-postfork-rename-open.png[Project details page: Rename Project button and Open in Code Editor button,50%,link={imagesDir}/project-details-postfork-rename-open.png]
+====
++
+.Image: Code Editor ...
+[%collapsible]
+====
+include::{includespath}/images-annotated.adoc[tag=editorsections]
+====
+.. Edit your source files.
++
+TIP: You can find your source files in the navigation pane under *Source Code*.
++
+... (Optional) If you chose a scaffold that still uses deprecated packages, migrate your project to use the `@withkoji/core` package. +
+Learn more: <<migrate-koji-core#_do_i_need_to_migrate,Do I need to migrate?>>
+... Install the `@withkoji/core` package.
```

Comment:
I think we should make it stand out somehow, because it's not a common noun. I figured this is the format they use when they include it in the code, so I used the code font.

Removed the backticks for now.